### ISSUE TITLE
Licensing

### ISSUE THREAD

**Issue by wordandahalf** [ZockerAxel/CrazyAdvancementsAPI#9]
What license is the code offered under?
Appreciate the time and effort you’ve put into this project!

**Comment by ZockerAxel** [comment]
The code is, as you can see, not published with a license, that means that all rights are reserved.
But i can assure you that you can do anything you want with my code, i don't really care, as long as you aren't a dick like publishing your own API on the spigot site and it's just a ripoff.

I don't add a license because it's just too much of a hassle. I had one and i was asked if i could change it because all projects that used my code had to have the same license and there is just so much stuff to know about licenses that i decided that i didn't want to have a license added from somewhere.

TLDR: Don't be a dick and I don't care what you do with it

**Comment by wordandahalf** [comment]
> The code is, as you can see, not published with a license, that means that all rights are reserved. But i can assure you that you can do anything you want with my code, i don't really care, as long as you aren't a dick like publishing your own API on the spigot site and it's just a ripoff.
> 
> I don't add a license because it's just too much of a hassle. I had one and i was asked if i could change it because all projects that used my code had to have the same license and there is just so much stuff to know about licenses that i decided that i didn't want to have a license added from somewhere.
> 
> TLDR: Don't be a dick and I don't care what you do with it

Appreciate it, thanks!

**Comment by HydrolienF** [comment]
If one day you change you mind about it there is simple licence that say "Do whatever you whant with my code"
cf https://choosealicense.com/licenses/unlicense/
